.github/workflows/ci.yml

```diff
@@ -13,11 +14,11 @@ jobs:
   CI:
     strategy:
       matrix:
-        architecture: [X64, ARM64]
-        distribution: [Ubuntu, AL2]
+        architecture: [ARM64]
+        distribution: [Amazon Linux 2]
```

Comment:
Is the intent here not to test X64 and Ubuntu combination?

Reply:
I'm testing stuff in my personal account's fork of this, but unfortunately github wants to keep automatically updating this PR. Sorry about that, you'll have to ignore the noise for the moment.